cmd/root.go

```diff
@@ -97,7 +150,7 @@ func GetJIRAClient(username, password, baseURL string) (*jira.Client, error) {
 		log.Errorf("Error connecting to JIRA; check your credentials. Error: %s", jira.CheckResponse(resp.Response))
 		return nil, jira.CheckResponse(resp.Response)
 	}
-	project = *proj
+	project = *(proj.(*jira.Project))
```

Comment:
Wouldn't a pointer to a pointer be `project = &(proj.(*jira.Project))`? That should dereference the pointer, so that `project` is just a jira.Project, no pointer.